test/tests/time_integrators/central-difference/tests

```diff
@@ -10,4 +9,12 @@
     requirement = "The CentralDifference time integrator shall correctly compute the first"
                   " and second time derivatives."
   [../]
+  [./ad_central_difference_dotdot]
+    type = 'CSVDiff'
+    input = 'ad_central_difference_dotdot.i'
+    csvdiff = 'ad_central_difference_dotdot_out.csv'
+
+    requirement = "The CentralDifference time integrator shall correctly compute the second"
+                  " time derivative of an AD variable."
```

Comment:
We should try to keep our requirements as implementation agnostic as possible, so probably something more like
```suggestion
    requirement = "A central differencing time integrator shall correctly compute the second"
                  " time derivative of an automatic differentiation variable."
```